app/access/tests/test_api.py

```diff
@@ -108,3 +106,82 @@ def test_get_users_returns_users_ordered_by_id(self):
                 },
             ]
         }
+
+    def test_post_users_creates_new_user_in_db_and_returns_it(self):
+
+        payload = {
+            "username": "donny",
+            "first_name": "Theodore Donald",
+            "last_name": "Kerabatsos",
```

Comment:
I didn't know that this was his real name 😂

Reply:
Me neither :smile: Not surprising, given that he has to "shut the f*** up" all the time :sweat_smile: